simulation/src/test/java/com/palantir/dialogue/core/SimulationTest.java

```diff
@@ -356,6 +364,7 @@ public void live_reloading() {
                 .requestsPerSecond(250)
                 .sendUntil(Duration.ofSeconds(10))
                 .clients(10, i -> strategy.getChannel(simulation, servers))
+                .abortAfter(Duration.ofMinutes(10))
```

Comment:
seems a bit generous no?

Reply:
I want it to be super clear on a graph if it timed out - remember this isn't real time, it'll take like 200ms

Reply:
👍